compiler-k2/build.gradle.kts

```diff
@@ -23,13 +23,18 @@ dependencies {
   api(project(":compiler-k2-api"))
   api(libs.kotlin.compiler)
 
+  implementation(project(":compiler-utils"))
+  implementation(libs.dagger2)
+
   kapt(libs.auto.service.processor)
 
   testCompileOnly(libs.auto.service.annotations)
 
   testImplementation(project(":compiler-testing"))
+  testImplementation(project(":compiler-utils"))
   testImplementation(libs.junit.jupiter.api)
   testImplementation(libs.kase)
   testImplementation(libs.kotest.assertions.api)
   testImplementation(libs.kotest.assertions.core.jvm)
+  testImplementation(libs.truth)
```

Comment:
😭 

Could you give the Kotest apis a shot?

Reply:
The majority of tests written within the last year have used Kotest assertions -- any time there was a new test class.  All assertions in new test tooling use Kotest assertions.  Going back to creating new test classes with Truth is an unnecessary step backwards.

Many of the existing tools make assumptions about hints, generated file names, KCT, and other K1 implementation details that don't really transfer over.  The allegories I'm writing in the K2 tooling do not make those assumptions, specifically so that they can have the interop we need.

Reply:
Updating for posterity: discussed offline with Rick and we agreed to focus on using Kotest for new test suites going forward to prioritize consistency across the K2 tests :+1: